### ISSUE TITLE
how can i trust netlify will stick around if i never pay for it?

### ISSUE THREAD

**Issue by sw-yx** [netlify/ask-netlify#35]
## About me

- **My name:** swyx
- **I've used Netlify:** lots

## My question in a little more detail

lots of startups die and i have trust issues. [Facebook killed Parse](https://en.wikipedia.org/wiki/Parse_(platform)). [Twitter acquired and killed Smyte](https://www.theregister.co.uk/2018/06/22/twitter_swallows_smyte_spits_out_customers/). How do I know you won't do the same?

**Comment by talves** [comment]
@sw-yx I asked @biilmann this question over a year and a half maybe. Still here after all my free sites!

You forgot: Adobe killed Flex (darnit!) 💀

**Comment by sw-yx** [comment]
haha just feeding them with questions

**Comment by talves** [comment]
Yeah, I know. Just ignore my add-ons. I should have been clearer with what I'm doing with my comments. Not all humor translates well.
